testkit/core/src/main/scala/com/lightbend/lagom/internal/testkit/TestConfig.scala

```diff
@@ -3,14 +3,29 @@
  */
 package com.lightbend.lagom.internal.testkit
 
-import com.typesafe.config.{ Config, ConfigFactory }
-
 private[lagom] object TestConfig {
-  private lazy val values =
-    s"""
-       |akka.persistence.journal.plugin = jdbc-journal
-       |akka.persistence.snapshot-store.plugin = jdbc-snapshot-store
-    """.stripMargin
+  def cassandraConfig(keyspacePrefix: String, cassandraPort: Int): Map[String, AnyRef] = Map(
```

Comment:
How is this related to updating to Play 2.7? Should we move this to another PR so that we can keep this one small?

Reply:
OK will do

Reply:
#1428